src/proto/boardctrl.proto

```diff
@@ -43,8 +43,15 @@ message ReceivedNote {
 message ListClientRequest {
 }
 
+message Client {
```

Comment:
feel free to add comments for new additions in `boardctrl.proto`, if we start to split into multi process the main components in the future, the interface will be reused quite